content/docs/ops/runbook/troubleshooting-bosh.md

```diff
@@ -83,69 +83,51 @@ the Troubleshooting Bosh Managed VMS section below.
 In order to troubleshoot a Bosh-managed VMS, you will need to first access a
 jumpbox, outlined previously.
 
-### Setting the deployment
+### Finding the deployment
 
-You'll need to set up `bosh` to the correct deployment. You can verify that the
-deployment is set by running the following command:
-
-```sh
-$ bosh status
-
-# ... output shortened for brevity
-
-Deployment
-  not set
-```
-
-You can get a list of all the current deployments by running the following
-command:
+In the pre-v2 BOSH CLI, you needed to set your deployment.
```

Comment:
We don't use the ruby bosh cli anymore. WDYT about dropping these notes and starting with `In order to see all the deployments...`?